services/ui-src/src/components/overlays/PlanComplianceTableOverlay.tsx

```diff
@@ -8,74 +14,188 @@ import {
   SortableTable,
   SaveReturnButton,
   BackButton,
+  OverlayContext,
 } from "components";
 // types
 import {
   EntityDetailsMultiformVerbiage,
   EntityDetailsTableContentShape,
   EntityShape,
   FormJson,
+  ScreenReaderOnlyHeaderName,
 } from "types";
 // utils
 import {
+  mapNaaarStandardEntity,
   mapNaaarStandardsData,
   NaaarStandardsTableShape,
 } from "components/tables/SortableNaaarStandardsTable";
+import { planComplianceStandardKey } from "../../constants";
 
 export const PlanComplianceTableOverlay = ({
+  analysisMethods,
   closeEntityDetailsOverlay,
   disabled,
-  entities,
   form,
   onChange,
   table,
   onSubmit,
   selectedEntity,
+  standards,
   submitting,
   validateOnRender,
   verbiage,
 }: Props) => {
-  const standardsTotalCount = entities.length;
-  // TODO: Set up counts
-  const exceptionsCount = 0;
-  const standardsCount = 0;
-
-  const [isEntityDetailsOpen, setIsEntityDetailsOpen] =
-    useState<boolean>(false);
+  const standardKeyPrefix = planComplianceStandardKey;
+  const { selectedStandard, setSelectedStandard } = useContext(OverlayContext);
 
   const DetailsOverlay = () => {
     const closeEntityDetailsFormOverlay = () => {
-      setIsEntityDetailsOpen(false);
+      setSelectedStandard(null);
+    };
+    let headRow = [] as ScreenReaderOnlyHeaderName[];
+    const bodyRows = [];
+    let formJson = JSON.parse(JSON.stringify(form));
```

Comment:
nit: could we use `structuredClone(form)` here? We use it in the [backend to copy form templates](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/blob/main/services/app-api/utils/formTemplates/formTemplates.ts#L282). When we do we have to [mock it in jest](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/blob/main/services/app-api/handlers/reports/create.test.ts#L29).

[docs on structuredClone](https://developer.mozilla.org/en-US/docs/Web/API/Window/structuredClone)